docs/data/data-grid/api-object/UseGridApiRef.tsx

```diff
@@ -20,7 +25,13 @@ export default function UseGridApiRef() {
     <Stack spacing={1} sx={{ width: '100%' }} alignItems="flex-start">
       <Button onClick={handleGoToPage1}>Go to page 1</Button>
       <Box sx={{ height: 400, width: '100%' }}>
-        <DataGrid {...data} apiRef={apiRef} pagination pageSize={10} />
+        <DataGrid
+          {...data}
+          apiRef={apiRef}
+          pagination
```

Comment:
Out of scop: `pagination` is forced to `true` in the MIT version. I assume it's a legacy from a demo with pro component
```suggestion
```